ci/buildspec_backend.yml

```diff
@@ -20,13 +20,13 @@ phases:
       - docker build backend/core -f build/docker/Dockerfile.backend --target test -t backend/core:test
       - docker run backend/core:test yarn test
       - docker build backend/core -f build/docker/Dockerfile.backend --target run -t backend/core:run-candidate
-      - docker tag backend/core:run-candidate "${ECR_ACCOUNT_ID}.dkr.ecr.${ECR_REGION}.amazonaws.com/${ECR_NAMESPACE}/backend:${CODEBUILD_RESOLVED_SOURCE_VERSION:0:8}-candidate"
-      - docker tag backend/core:run-candidate "${ECR_ACCOUNT_ID}.dkr.ecr.${ECR_REGION}.amazonaws.com/${ECR_NAMESPACE}/backend:run-candidate"
       - docker build backend/core -f build/docker/Dockerfile.backend --target migrate -t backend/core:migrate-candidate
+
       - docker tag backend/core:migrate-candidate "${ECR_ACCOUNT_ID}.dkr.ecr.${ECR_REGION}.amazonaws.com/${ECR_NAMESPACE}/backend:migrate-candidate"
+      - docker tag backend/core:run-candidate "${ECR_ACCOUNT_ID}.dkr.ecr.${ECR_REGION}.amazonaws.com/${ECR_NAMESPACE}/backend:${CODEBUILD_RESOLVED_SOURCE_VERSION:0:8}-candidate"
```

Comment:
The hash on the tag is enough for uniqueness, but to anyone looking at images in the repo it isn't clear what this image is a candidate for.  Something like `run-${CODEBUILD_RESOLVED_SOURCE_VERSION:0:8}` would keep it readable and descriptive.  Since that tag will never need to change, we don't need the "candidate" part.